src/memory-analyzer/analyze_symbol.cpp

```diff
@@ -190,7 +190,8 @@ exprt symbol_analyzert::get_char_pointer_value(
   }
   else
   {
-    return it->second;
+    return symbol_exprt{to_symbol_expr(it->second).get_identifier(),
```

Comment:
comment? if you had to understand something - please tell us what and why. 
This should greatly help in reducing review time and increase accuracy of reviews.

Reply:
Done, or maybe removed.